canonical-web-and-design/ubuntu.com
# Update geolocation pdf

## Done

- Created a pdf of [this document](https://docs.google.com/document/d/1qesV0RGWxv1lkK8K1hEprwQqTpiIcLhluJVFdT__iLE/edit?ts=5e972bd3) and uploaded it
- Updated the link on http://0.0.0.0:8001/legal/data-privacy#where-we-transfer-and-store-your-information
- Updated the [copy doc](https://docs.google.com/document/d/1AlOE2_-BqZNP4hiiuf8kZney8g1zvqAlDYl2u3X6m6I/edit#)
- Request per Katherine O

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/legal/data-privacy#where-we-transfer-and-store-your-information
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the link is updated and the pdf is the one dated today
